docs/api-reference/next.config.js/rewrites.md

```diff
@@ -360,6 +412,10 @@ module.exports = {
 When leveraging [`i18n` support](/docs/advanced-features/i18n-routing.md) with rewrites each `source` and `destination` is automatically prefixed to handle the configured `locales` unless you add `locale: false` to the rewrite. If `locale: false` is used you must prefix the `source` and `destination` with a locale for it to be matched correctly.
 
 ```js
+/**
```

Comment:
Do we want to add these everywhere in the docs? While it's definitely useful, could we just mention it once in the `next.config.js` settings if people want to opt-in?